Documentation/ApiOverview/Rte/InTheFrontend/Introduction.rst

```diff
@@ -0,0 +1,116 @@
+.. include:: /Includes.rst.txt
+.. index::
+   pair: Rich text editor; Frontend
+.. _rte-frontend-introduction:
+
+==================================================
+Including a Rich Text Editor (RTE) in the frontend
+==================================================
+
+When you add forms to a website you might want to offer
+formatting options like bold, italic etc.. Rich Text Editors offer
+extensive options that are configurable for individual needs.
+
+This chapter outlines conceptual and technical information
+about adding an RTE on frontend pages.
+
+.. figure:: ../Images/RteBackend.png
+   :alt: A RTE in the TYPO3 BE
+
+   The rtehtmlarea RTE activated in the TYPO3 backend
+
+.. tip::
+
+   This description about CKEditor includes examples and common challenges for
+   the frontend.
+   
+   You can use other editors with TYPO3 and some points, like
+   handling of data on the server, are independent of the distinct editor
+   in the frontend.
+
+.. note::
```

Comment:
I would not include this note. Such a documentation would most likely belong in the extensions documentation. rte_ckeditor is currently the only one provided by the core

Reply:
You marked the text above the note, thats a bit confusing, but I refer to the note below that part.
For the frontend there is no need to use CKEditor, it's completely independent.  
In the manual of CKEditor I think this info is lost, it never belongs there in my opinion.  
Nevertheless, I could drop that note completely, people who feel the need for it or want to do something find a way to articulate it.

Reply:
unintentionally it was already gone before, no clue what happend